tools/memap.py

```diff
@@ -10,10 +10,11 @@
 import argparse
 from prettytable import PrettyTable
 
-from tools.utils import argparse_filestring_type, \
+from utils import argparse_filestring_type, \
     argparse_lowercase_hyphen_type, argparse_uppercase_type
 
 DEBUG = False
+DETAILED = False
```

Comment:
*args* cannot be accessed from inside the *path_object_to_module_name* so unless I add another parameter to  *path_object_to_module_name* signature and propagate it all the way up to the main I don't see how to implement this without a global variable. 

Is this what you would like me to implement? Is there an easier solution? 

It is like the 5th time or so that I code something in python so I'd appreciate the input :)